src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/Sentence.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
+
+import java.time.LocalDate
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Sentence as IntegrationApiSentence
+
+data class Sentence(
+  val sentenceDetail: SentenceDetail?,
+) {
+  fun toSentence(): IntegrationApiSentence = IntegrationApiSentence(
+    startDate = this.sentenceDetail?.sentenceStartDate,
+  )
+
+  data class SentenceDetail(
```

Comment:
I'd probably put this in its own file as we've done for ImageDetail and OffenceHistoryDetail just to avoid confusion